### DISCUSSION TITLE
Content delivery snapshot history

### DISCUSSION THREAD

**Discussion by llucmk** [tolgee/tolgee-platform#2674] [Ideas]
Has anybody came across this issue or lack of feature? For quality control and production environments would be nice to have a history of content delivery publishes. 

Reasons for this feature would be in the case there's a mistake in the content delivery and to rollback safely to the previous snapshot done that was stable. Of course this could be addressed with QA before the publishing, but you never know.

![image](https://github.com/user-attachments/assets/bcdab169-28bb-46f8-b696-55960d08b783)

**Comment by JanCizmar** [comment]
Hey! That's a good suggestion. We will discuss this with the team! 🚀 @ZuzanaOdstrcilova @stepan662
